.readme.templ

```diff
@@ -0,0 +1,42 @@
+![NATS](large-logo.png)
+
+# NATS Architecture And Design
+
+This repo is used to capture architectural and design decisions as a reference of the server implementation and expected client behavior.
+
+# Architecture Decision Records
```

Comment:
Suggestion: add a table listing all ADR not in a final ("Implemented", "Rejected") state. Below some seem to still be in Approved state, but are done IIRC.

Reply:
ok, will see if I can add it in a followup - keen to get merged so others can add issues and ADRs can come here.

We do need to be better at updating ADRs to Implemeneted.